Jenkinsfile_CNP

```diff
@@ -2,7 +2,7 @@
 
 @Library("Infrastructure") _
 
-def yarnBuilder = new uk.gov.hmcts.contino.YarnBuilder(this)
+yarnBuilder = new uk.gov.hmcts.contino.YarnBuilder(this)
```

Comment:
Just curious on the reason for this change?

Reply:
error! fixed now.